specs/spec_reader.go

```diff
@@ -83,9 +83,10 @@ func (r *SpecReader) validate() error {
 			if r.Destinations[destination] == nil {
 				return fmt.Errorf("source %s references unknown destination %s", source.Name, destination)
 			}
-			if destinationSourceMap[destination] == "" {
-				destinationSourceMap[destination] = source.Path + "@" + source.Version
-			} else if destinationSourceMap[destination] != source.Path+"@"+source.Version {
+			destinationToSourceKey := fmt.Sprintf("%s-%s", destination, source.Path)
```

Comment:
It's ok to `destination` name and not `destination.Path` as those destination names are unique by definition in a single sync